diff --git a/source/str_util.c b/source/str_util.c
index 90b665f..c461e4b 100644
--- a/source/str_util.c
+++ b/source/str_util.c
@@ -97,7 +97,7 @@ static PyObject * get_byte_encoding(PyObject *self, PyObject *args)
         return Py_BuildValue("s", "wide");
     if (byte_encoding == ENC_NARROW)
         return Py_BuildValue("s", "narrow");
-    return Py_None; // should never happen
+    Py_RETURN_NONE; // should never happen
 }
 
 
@@ -127,7 +127,7 @@ static PyObject * set_byte_encoding(PyObject *self, PyObject *args)
         return NULL;
     }
     
-    return Py_None;
+    Py_RETURN_NONE;
 }
 
 
